ui/dashboards/src/context/TemplateVariableProvider/TemplateVariableProvider.tsx

```diff
@@ -23,6 +24,7 @@ import {
   DEFAULT_ALL_VALUE as ALL_VALUE,
 } from '@perses-dev/plugin-system';
 import { VariableName, VariableValue, VariableDefinition } from '@perses-dev/core';
+import { useVariableQueryParams, getInitalValuesFromQueryParameters, getURLQueryParamName } from './query-params';
```

Comment:
I like how you organized this, I put a lot of the time-range implementation inside of utils, which I'm going to change now. For disabling params, we can do it in QueryParamProvider like you mentioned, but is the plan for TemplateVariableProvider to also have a new prop for this?

Reply:
Yeah exactly. I didn't implement it yet, but it would basically be:
```tsx
<TemplateVariableProvider enabledURLParams={false} {...rest} />
```

The top level Dashboard component can control that.